src/data/faq.json

```diff
@@ -554,11 +554,13 @@
                                 ]
                             },
                             {
-                                "title": "I receive a notification that the validity of a certificate is about to expire. What do I have to do?",
+                                "title": "How and when can certificates be reissued in the app?",
                                 "anchor": "dcc_replacement_howto",
                                 "active": false,
                                 "textblock": [
-                                    "The technical validity of COVID digital certificates expires automatically after 365 days. For all relevant certificates, you will receive a notification about this in the Corona-Warn-App 28 days before expiration. In order to continue to be able to prove your vaccination status with the app, the relevant certificates must be updated. You will soon be able to carry out the necessary reissue yourself in the app with just a few clicks. See also <a href='#glossary_validity'>\"Validity of an EU Digital COVID Certificate\"</a>."
+                                    "If you receive a corresponding message in the Corona-Warn-App, you can voluntarily request a new vaccination certificate or recovery certificate free of charge via the app. After you have given your consent, the app transmits the certificate or several certificates for verification to the RKI, which issues an updated certificate and transmits it to your app. You can then find the updated certificate in the Certificates tab.",
+                                    "<ul> <li>The old certificate is automatically moved to the trash and deleted after 30 days. The old certificate can be restored at any time within the 30 days.</li> <li>The possibility of reissue begins 28 days before the expiration of the certificate and ends 90 days after the expiration of the certificate</li> <li><strong>IMPORTANT</strong>: The reissue can be made maximum three times within this period. Because of this limitation, we recommend that you export the reissued certificate as a PDF file and print it out so that you can scan the QR code of the certificate again if necessary. (see &quot;<a href='#eu_dcc_export' target='_blank'>How do I create a printout of my EU Digital COVID Certificate?</a>&quot;</li> </ul>",

```

Comment:
"The reissue can be made maximum three times within this period." =>

"The reissue can be done a maximum of three times within this period."

which is better grammar. 

Alternatively, because "do or make a reissue" sounds a bit clumsy,:

"A certificate can be reissued a maximum of three times within this period."

Reply:
The reissue can be done a maximum of three times within this period.